backend/src/main/java/c/team/session/controller/SessionManagerController.java

```diff
@@ -52,4 +68,15 @@ public void closeSession(@RequestBody SessionCloseRequest request){
     public final ResponseEntity<Error> handleException(SessionNotFoundException ex){
         return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
     }
+
+    @ExceptionHandler(SessionNotOwnedException.class)
+    public final ResponseEntity<Error> handleException(SessionNotOwnedException ex){
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
+    }
+
+    // Duplicate from UserAccountController, probably a universal exception handler can be created
+    @ExceptionHandler(UsernameNotFoundException.class)
+    public final ResponseEntity<Error> handleException(UsernameNotFoundException ex){
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
```

Comment:
NO_CONTENT 204 to pozytywny response code, wskazujący na to że wszystko się udało. W tym przypadku jednak zwracałbym coś innego, na przykład 403 FORBIDDEN

Reply:
O dobra, faktycznie - znajdę coś innego, ale 403 FORBIDDEN ma ten problem, że gdy wystąpi jakikolwiek inny wyjątek, który nie jest handlowany przy pomocy @ExceptionHandler to też http zwraca 403. Pasuje idealnie, ale jeśli coś przeoczymy to trochę będzie ciężko rozróżnić co się stało.